src/main/java/com/alinab/taskOneGOFpatterns/behavioral/chainOfResponsibility/MultiplyNumbersHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.alinab.taskOneGOFpatterns.behavioral.chainOfResponsibility;
+
+public class MultiplyNumbersHandler implements Handler {
+
+    private Handler nextHandler;
+
+    @Override
+    public void setNextHandler(Handler nextHandler) {
+        this.nextHandler = nextHandler;
+    }
+
+    @Override
+    public void calculate(Numbers request) {
+        if (request.getCalculationRequest().equals("mult")) {
+            System.out.println(request.getNumberOne() + " * " + request.getNumberTwo() + " = "
```

Comment:
Make this string constant with naming and move constant on the first place while comparing. It will help to avoid NPE
"mult".equals(request.getCalculationRequest())

Reply:
Done